### PR TITLE
CAPI-1210 : Follow up PR

### PR DESCRIPTION
**Pull request by SiniWilson**

### CODE REVIEW

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh [hunk 1]
```diff
@@ -51,30 +51,54 @@ sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack
...
+			  "SERVICE_STATUS":{"S":"active"},
+			  "SERVICE_METADATA":{"M":{
+							  "securityGroupIds":{"S":"'$securityGroupIds'"},
+							  "subnetIds":{"S":"'$subnetIds'"},
+							  "iamRoleARN":{"S":"'$iamRoleARN'"},
+							  "providerMemorySize":{"S":"256"},
+							  "providerRuntime":{"S":"nodejs4.3"},
+							  "providerTimeout":{"S":"160"},
+							  "runtime":{"S":"nodejs"},
+							  "type":{"S":"api"}
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: cloud-logs-streamer is a lambda function

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh [hunk 2]
```diff
@@ -51,30 +51,54 @@ sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack
 sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack/serverless-python.yml
 
 #-------------------------------------------
-platform_services=("cognito-authorizer" "logs" "usermanagement" "services-handler" "events" "services" "logout" "login" "cloud-logs-streamer" "is-service-available" "delete-serverless-service" "create-serverless-service" "email" )
+platform_services=("cognito-authorizer" "platform_logs" "platform_usermanagement" "platform-services-handler" "platform_events" "platform_services" "platform_logout" "platform_login" "cloud-logs-streamer" "is-service-available" "delete-serverless-service" "create-serverless-service" "platform_email" )
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: this is incorrect. The domain is platform and service name is logs.

Comment (SiniWilson) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: In cooperated the review comment

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh (hunk 1 above)
Comment (SiniWilson) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: In cooperated the review comment

**Pull request merged**